src/project/internal/projectmigrator.cpp

```diff
@@ -55,6 +60,241 @@ static MigrationType migrationTypeFromMscVersion(int mscVersion)
     return MigrationType::Unknown;
 }
 
+static int repitchMdlSnares(int pitch)
```

Comment:
It would be better to add a separate class for MDL-related conversions and move all this code into it. This class will have 1 static public function - remapPercussion(score) and we can name it as MdlMigrator

Reply:
`remapPercussion(score)` can't be static because of the call to [`globalConfiguration()`](https://github.com/musescore/MuseScore/pull/23400/files#diff-6ae65b9b2b39943ee5bd8ecee79db611d15f06d6ffad2ea52417fd075656b70dR67).

The [other functions](https://github.com/musescore/MuseScore/pull/23400/files#diff-1e418392c5d201c3b01226f6f40fa8df90f261a5f21438fbeee5c46517a98e95R47-R50) can be static. The [repitch functions](https://github.com/musescore/MuseScore/pull/23400/files#diff-1e418392c5d201c3b01226f6f40fa8df90f261a5f21438fbeee5c46517a98e95R53-R57) have to be static to enable them to be [passed as variables](https://github.com/musescore/MuseScore/pull/23400/files#diff-6ae65b9b2b39943ee5bd8ecee79db611d15f06d6ffad2ea52417fd075656b70dR171).